### PR TITLE
[New] Add jQuery.x18n@1.1.3 w/ git auto-update

### LINKED ISSUE #8011: [Request] Add x18n series libs
~~https://github.com/florian/x18n~~
https://github.com/marco-a/react-x18n
https://github.com/florian/jQuery.x18n/
~~https://github.com/SBoudrias/handlebars-x18n~~

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #8011
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/florian/jQuery.x18n
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 4
   - Count of stars: 31
   - Count of forks: 1
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.x18n/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "filename": "jquery.x18n.min.js",
+  "keywords": [
+    "jquery",
+    "x18",
+    "javascript",
+    "internationalisation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/florian/jQuery.x18n.git"
```
Comment (sufuf3) [inline ajax/libs/jquery.x18n/package.json]: Let's use `https`.
Thanks.

File: ajax/libs/jquery.x18n/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.x18n",
+  "version": "1.1.3",
+  "description": "A JavaScript internationalisation library",
```
Comment (sufuf3) [inline ajax/libs/jquery.x18n/package.json]: Let's use this description https://github.com/florian/jQuery.x18n/blob/master/jQuery.x18n.jquery.json#L4
This description is mentioned `x18n` part, which I think is good than current description.
What do you think?

Thanks.

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sufuf3** [comment]
@sashberd I saw your commit messages which are in the past was written `... git auto-update` or `... npm auto-update`, but this PR's commit message is `... git autoupdate`.
Although, this PR is good enough, I still think we can make this commit message better.
Because IMO, if we use the same format of commit message, when someone search some keyword of the past commit, the good commit message is helpful to everyone to get what they want.

So, what do you think?
Thanks.

And this adding-lib's issue is opened by @PeterDaveHello , but the lib doesn't have enough notable popularity. 
Maybe this PR need to be discuss.
@PeterDaveHello could you tell us how to do for this lib? Could we host this lib?
Many thanks.

**Comment by PeterDaveHello** [comment]
Wrong PR title and commit message @sufuf3 @sashberd

File: ajax/libs/jquery.x18n/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "name": "jquery.x18n",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
+  "license": "MIT",
+  "filename": "jquery.x18n.min.js",
+  "keywords": [
+    "jquery",
+    "x18",
+    "javascript",
+    "internationalisation"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.x18n/package.json]: I think we missed one important keyword?
https://github.com/florian/jQuery.x18n/blob/master/jQuery.x18n.jquery.json#L6
and maybe also `internationaliziation`?

Comment (PeterDaveHello) [inline ajax/libs/jquery.x18n/package.json]: I think we missed one important keyword?
https://github.com/florian/jQuery.x18n/blob/master/jQuery.x18n.jquery.json#L6
and maybe also `internationaliziation`?

**Comment by sashberd** [comment]
@PeterDaveHello You are right man, my false... The PR was updated. FYR

**Comment by PeterDaveHello** [comment]
@sashberd please do not request the review from me directly, I think @sufuf3 will do it, thanks.

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

File: ajax/libs/jquery.x18n/package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.x18n",
+  "filename": "jquery.x18n.min.js",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
+  "author": {
+    "name": "Florian H.",
+    "url": "https://github.com/js-coder"
```
Comment (extend1994) [inline ajax/libs/jquery.x18n/package.json]: Though author is not an important info for cdnjs now, I believe this should be https://github.com/florian

File: ajax/libs/jquery.x18n/package.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.x18n",
+  "filename": "jquery.x18n.min.js",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
```
Comment (extend1994) [inline ajax/libs/jquery.x18n/package.json]: Seems like `internationalisiation` from upstream is a mistake of "internationalisation". In description of [README.md](https://github.com/florian/jQuery.x18n#jqueryx18n-) the latter one is used.

File: ajax/libs/jquery.x18n/package.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  "author": {
+    "name": "Florian H.",
+    "url": "https://github.com/js-coder"
+  },
+  "keywords": [
+    "i18n",
+    "jquery",
+    "x18",
+    "javascript",
+    "internationalization"
```
Comment (extend1994) [inline ajax/libs/jquery.x18n/package.json]: So maybe one more "internationalisation".

File: ajax/libs/jquery.x18n/package.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "https://github.com/florian/jQuery.x18n.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/florian/jQuery.x18n.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "jquery.x18n.js"
```
Comment (extend1994) [inline ajax/libs/jquery.x18n/package.json]: Let's use more flexible one? Such as `jquery.x18n!(*.coffee)`?

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

File: ajax/libs/jquery.x18n/package.json (hunk 7 above)
Comment (sufuf3) [reply ajax/libs/jquery.x18n/package.json]: I thought we use included mode when `lib` as the basePath, `!` is excluded mode.
I am a little confuse in this pattern.
Using `!` as excluded mode is good when `lib` as the basePath?
Thanks

Comment (extend1994) [reply ajax/libs/jquery.x18n/package.json]: Sorry for the mistake. @sashberd Please use `jquery.x18n*.+(js|css|map)`. Thanks.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@PeterDaveHello The lin name was updated, PR was updated. FYR

**Comment by sashberd** [comment]
@PeterDaveHello ping

**Pull request merged**